src/Mailer/Transport/SimpleQueueTransport.php

```diff
@@ -49,6 +49,12 @@ public function send(Email $email) {
 			'template' => $email->template(), //template() gives 2 values - template and layout
 			'viewVars' => [$email->viewVars()]
 		];
+		
+		foreach ($settings as $setting => $value) {
+			if (empty($value) || array_key_exists(0, $value) && empty($value[0])) {
```

Comment:
I dont like empty() checks as it could false positive '0' etc.
So if we can be more strict that would be good.
Also, should we check on is_array($value) first before treating it as such?